src/core/agent/services/utils.ts

```diff
@@ -45,13 +45,14 @@ export const OnlineOnly = (
       const executeResult = await originalMethod.apply(this, args);
       return executeResult;
     } catch (error) {
-      const errorStack = (error as Error).stack;
+      const errorStack = (error as Error).stack as string;
```

Comment:
Why do we still use `.stack` here? Does `.message` not work everywhere?

Reply:
This is used for android and the web version. The ios version gives us another message, so I think we can still use stack here.

Reply:
Hmm, and just here? So for e.g. `getConnectionById` android/web and iOS are the same?

Reply:
Yeah, it only has a difference when the KERIA goes offline. However, I will change this to using `message` for consistency.